src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -284,6 +284,52 @@ public void ReuseAddress_Windows(bool? exclusiveAddressUse, bool? firstSocketReu
             ReuseAddress(exclusiveAddressUse, firstSocketReuseAddress, secondSocketReuseAddress, expectFailure);
         }
 
+        [Fact]
+        public void BindDuringTcpWait()
+        {
+            int port = 0;
+            using (Socket a = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
+            {
+                a.Bind(new IPEndPoint(IPAddress.Loopback, 0));
+                port = (a.LocalEndPoint as IPEndPoint).Port;
+                a.Listen(10);
+
+                // Connect a client
+                using (Socket client = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
+                {
+                    client.Connect(new IPEndPoint(IPAddress.Loopback, port));
+                    using (Socket acceptedClient = a.Accept())
+                    { }
+                }
+            }
+
+            // Bind a socket to the same address we just used.
+            using (Socket b = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
+            {
+                b.Bind(new IPEndPoint(IPAddress.Loopback, port));
+            }
+        }
+
+        [Fact]
+        public void ExclusiveAddressUseTcp()
```

Comment:
What is this test doing differently from https://github.com/dotnet/corefx/blob/2ad41da209006b99878beaf09f50b8c4bbd49f9a/src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs#L239?

Reply:
This one is using TCP.

Reply:
I see.  Ok.